demisto_sdk/commands/find_dependencies/tests/find_dependencies_test.py

```diff
@@ -1,234 +1,289 @@
+import json
+import os
+import shutil
+from pathlib import Path
+from typing import List
+
 import demisto_sdk.commands.create_id_set.create_id_set as cis
 import networkx as nx
 import pytest
 from demisto_sdk.commands.find_dependencies.find_dependencies import \
     PackDependencies
+from TestSuite.repo import Repo
 from TestSuite.test_tools import ChangeCWD
 from TestSuite.utils import IsEqualFunctions
 
 
-@pytest.fixture()
-def id_set(repo):
+def create_a_pack_entity(pack, entity_type: str = None, entity_id: str = None, entity_name: str = None,
+                         commands: List[str] = None):
+    """
+    Given
+        - A Pack.
+
+    When
+        - add an entity to the pack.
+
+    Then
+        - Adds the entity to the pack with basic data.
+    """
+    if entity_type == 'script':
```

Comment:
I would suggest creating an enum for it, it's always better to have pre-defined values in those cases.